lib/testsuite/index.js

```diff
@@ -310,7 +310,7 @@ class TestSuite {
         results: this.reporter.exportResults()
       }));
     } 
-    this.client.transport.testSuiteFinished(failures);
+    this.client.transport.testSuiteFinished(failures, this.reporter.testResults.lastError);
```

Comment:
This is an `Error` object; it does get stringified automatically, but it contains a lot of junk (like the ASCII colour escape codes and some additional properties) which shouldn't be there. Why not send the `lastError.message`? I

Reply:
But I think that `lastError.message` also has the control chars?